src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/ServicesApiControllerIntegrationTest.java

```diff
@@ -484,22 +484,45 @@ public void addAccessRights() {
         assertEquals(calculatePrimeClientsBefore + 3, updatedServiceClients.size());
     }
 
-    @Test(expected = ConflictException.class)
```

Comment:
Missing `@Test` annotation so the test does not actually run

Reply:
Oops. Added.